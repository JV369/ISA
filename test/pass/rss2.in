./feedreader http://www.mocky.io/v2/5bd58738310000600041db06 -aT -u
