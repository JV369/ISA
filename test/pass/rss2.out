*** TestOfRSS ***
Test RSS2
Autor: JV
Aktualizace: Tue, 27 May 2003 08:37:32 GMT
URL: http://www.test.com

Test RSS1
Autor: JV
Aktualizace: Tue, 20 May 2003 08:56:02 GMT
URL: http://www.test1.com

